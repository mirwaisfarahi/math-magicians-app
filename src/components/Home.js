import React from 'react';
import { Container, Row, Col } from 'reactstrap';
import math from '../images/math.png';

const Home = () => (
  <div className="home">
    <h3 className="heading">Welcome to our Page!</h3>
    <Container>
      <Row>
        <Col xs="8">
          <p>
            &quot;Mathematics is the Queen and the Servant of the Sciences.&quot; I think this is
            a great way to think of the scope and power of mathematical thinking.
            To me, mathematics underlies the sciences, and that statement is
            only becoming more true as time passes.
          </p>
        </Col>
        <Col className="img">
          <img width="90%" xs="4" src={math} alt="Math" />
        </Col>
      </Row>
    </Container>
  </div>
);

export default Home;
