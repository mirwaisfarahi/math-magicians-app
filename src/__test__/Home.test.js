import React from 'react';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom';
import Home from '../components/Home';

describe('test calculator function', () => {
  it('Match Home DOM', () => {
    const homeTree = render(<Home />);
    expect(homeTree).toMatchSnapshot();
  });
});
