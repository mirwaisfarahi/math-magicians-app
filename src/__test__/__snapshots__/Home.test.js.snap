// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test calculator function Match Home DOM 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="home"
      >
        <h4
          class="heading"
        >
          Welcome to our Page!
        </h4>
        <div
          class="container"
        >
          <div
            class="row"
          >
            <div
              class="col-8"
            >
              <p>
                "Mathematics is the Queen and the Servant of the Sciences." I think this is a great way to think of the scope and power of mathematical thinking. To me, mathematics underlies the sciences, and that statement is only becoming more true as time passes.
              </p>
            </div>
            <div
              class="img col"
            >
              <img
                alt="Math"
                src="math.png"
                width="90%"
                xs="4"
              />
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="home"
    >
      <h4
        class="heading"
      >
        Welcome to our Page!
      </h4>
      <div
        class="container"
      >
        <div
          class="row"
        >
          <div
            class="col-8"
          >
            <p>
              "Mathematics is the Queen and the Servant of the Sciences." I think this is a great way to think of the scope and power of mathematical thinking. To me, mathematics underlies the sciences, and that statement is only becoming more true as time passes.
            </p>
          </div>
          <div
            class="img col"
          >
            <img
              alt="Math"
              src="math.png"
              width="90%"
              xs="4"
            />
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
